Lib/test/libregrtest/run_workers.py

```diff
@@ -79,8 +79,12 @@ class MultiprocessResult:
     err_msg: str | None = None
 
 
+class WorkerExited:
```

Comment:
"worker" can be a child process or a thread, I would prefer to be extra explicit here:

```suggestion
class WorkerThreadExited:
```